newIDE/app/src/InstancesEditor/SelectionRectangle.js

```diff
@@ -13,6 +14,7 @@ export default class SelectionRectangle {
   _instancesInSelectionRectangle: gdInitialInstance[];
 
   selector: gdInitialInstanceJSFunctor;
+  _temporaryRectangle: Rectangle;
```

Comment:
For temporaries, it's less important for the IDE than in the runtime, but let's have a comment explaining where they are used.

Reply:
Or just renaming it to `_temporaryAABB` (because "Rectangle" is just the name of the type, this does not give information about what this temporary is used for).